### PR TITLE
docs: llamaindex search and retrieval tutorial

### PR DESCRIPTION
**Pull request by axiomofjoy**
resolves #734

### CODE REVIEW

File: tutorials/llama_index_search_and_retrieval_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,819 @@
+{
```
Comment (mikeldking) [inline tutorials/llama_index_search_and_retrieval_tutorial.ipynb:1]: <p>might link to documentation as to how you wold do this via llama_index docs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (mikeldking) [inline tutorials/llama_index_search_and_retrieval_tutorial.ipynb:1]: <p>Emoji is tad confusing. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (mikeldking) [inline tutorials/llama_index_search_and_retrieval_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #11.    print("Embedding Dimension")
</pre><p>do we need this print?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (mikeldking) [inline tutorials/llama_index_search_and_retrieval_tutorial.ipynb:1]: <p>might add why GPT4 is better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

**Review by mikeldking — APPROVED** [review]
Nice one!

File: tutorials/llama_index_search_and_retrieval_tutorial.ipynb (hunk 1 above)
Comment (axiomofjoy) [reply tutorials/llama_index_search_and_retrieval_tutorial.ipynb:1]: <p>I'll show the embedding itself. I want to call attention to the fact that the embeddings are available in addition to the query and response.</p>

Comment (axiomofjoy) [reply tutorials/llama_index_search_and_retrieval_tutorial.ipynb:1]: <p>Easy to miss this detail.</p>

Comment (axiomofjoy) [reply tutorials/llama_index_search_and_retrieval_tutorial.ipynb:1]: <p>Changed to 💬</p>

Comment (axiomofjoy) [reply tutorials/llama_index_search_and_retrieval_tutorial.ipynb:1]: <p>👍</p>

Comment (axiomofjoy) [reply tutorials/llama_index_search_and_retrieval_tutorial.ipynb:1]: <p>Good call</p>

**Pull request merged**